src/master-playlist-controller.js

```diff
@@ -276,30 +283,6 @@ export class MasterPlaylistController extends videojs.EventTarget {
           this.audioSegmentLoader_.load();
         }
       }
-
-      if (!updatedPlaylist.endList) {
```

Comment:
Presumably we no longer have to update `seekable` or deal with `durationchange` since we are using a native media source. This code should have lived in our non-native media source I think.